lib/prometheus/client/histogram.rb

```diff
@@ -94,6 +94,16 @@ def values
         end
       end
 
+      def init_label_set(labels)
+        base_label_set = label_set_for(labels)
+
+        @store.synchronize do
+          [*@buckets, "+Inf", "sum"].each do |bucket|
```

Comment:
Can we do `(@buckets + ["+Inf", "sum"])`?
I find this splat harder to read

Reply:
Yeah sure!